src/wwwroot/js/site.js

```diff
@@ -2,3 +2,41 @@
 // for details on configuring this project to bundle and minify static web assets.
```

Comment:
Really nice that you moved javascript into separate file since now we don't have to repeat same code and overall better structure.